### PR TITLE
Legacy Synthetic Hardware quirk (Makes synth/IPC mechanics opt-in)

### PR DESCRIPTION
**Pull request by deathride58**
## About The Pull Request
This PR does exactly what it says on the tin. This makes it so that the species traits associated with synths and IPCs (healing from welding tools + cable coil, surgery requiring standard tools, toxloss being replaced with system corruption, etc), are instead tied to an opt-in quirk, dubbed "Legacy Synthetic Hardware". This is the implementation I originally promised long before unique IPC mechanics were actually implemented, and requested several times following the implementation. So here I am, fulfilling that small promise I've made several times over the past three years. This allows any arbitrary species to take on the traits associated with IPCs/synths, while also allowing people to play IPC/synth characters without being forced to be subject to the mechanics. Ingame, this quirk is flavored as a generational hardware difference, in which older synthetic hardware lacks complete parity with organic beings.

This PR also allows for some weird abominations, such as legacy synthetic plasmamen, and legacy synthetic slimes. Synth mechanics should, in theory, not conflict with the mechanics of any species (and some surface-level testing suggests that the combination does indeed work just fine), but it's bound to make for some fairly interesting gameplay moments for those that choose to opt in to that wild ride. 

## Why It's Good For The Game
My early talking of this PR has garnered near-unanimous positive reception across the board. This allows the current synth mechanics to still exist, but in a way where players aren't strictly forced to be subject to them if they want to play a synthetic character. This will allow the mechanics to become even more experimental, in a way where players can simply opt out if they don't like them. If more positives are added to synthetics, then the perk's cost can be adjusted accordingly.

## Changelog
:cl: Bhijn
add: Added the Legacy Synthetic Hardware quirk. This quirk applies the mechanics associated with the current implementation of IPCs and synths, to any arbitrary species on spawn. This means you can play a bootleg protean if you want.
del: IPCs, synthlizards, and anthropomorphic synthetics, are all no longer inherently tied to the traits of synthetics.
tweak: Examine text now shows an indication of the legacy synthetic hardware quirk, in the form of a message stating that the person's movements appear rather artificial.
tweak: Medical scanners now have two additional info bits: "Treatment Complications", and "Recommended Department Involvement". This will display all possible things that can potentially affect medical treatment for a patient, and also displays the recommended departments that should be involved in the patient's treatment. "Treatment Complications" does not show up if a patient doesn't have anything that could complicate treatment, but "Recommended Department Involvement" will always show, being "Medical" by default. All things listed in these sections are color-coded for easy recognition while skimming.
/:cl:

### CODE REVIEW

**Comment by MrJWhit** [comment]
Hang on, how can you differentiate between the two in-game?
Just slam your lit welder on every synth that walks into medbay just to be safe?

**Comment by lolman360** [comment]
should be a neutral quirk tbh

**Comment by ghost** [comment]
meat synths are cringe

**Comment by LordSaladin** [comment]
Slime synths, also this just makes the problems of current synths a "quirk" ~~because we clearly said that we wanted more quirks~~

**Comment by R3dtail** [comment]
This is awesome.   I understand there was a LOT of salt in the discord about this PR for...some reason but.  Ultimately, this  helps avoid a LOT of headaches for players and developers alike.  Players can pick which form they want and play the game either as a somewhat modified human, or a fullon legacy synthetic with all the experimental features that developers are free to play with and tinker as they see fit and wind up better equipped to receive feedback since players are actually able to make an educated decision of what they're getting into regarding the trait.  Also

For the trait text:  Can you add that since this code is highly experimental, features can potentially change quite frequently?  Helps give more information to the player

**Comment by NicoDeimos** [comment]
Against this, goes against the point of having unique racial traits where slimepeople, mushrooms, plasmamen etc exist without the same need for a snowflake quirk. the move to make IPCs mechanically act like robots falls in line with those other racial ideals. It's a relatively recent change if I recall and will obviously have its kinks to work through, and obvious weaknesses and strengths like other races exhibit. As far as I'm aware there's only been one or two synth players who've been especially vocal about not liking the race they choose to keep playing and one or two people who were against (making it a quirk). "Almost unanimous" is disingenuous when the sample size is that low, unless you happen to have a more exhaustive list for us to reference.

**Comment by MrJWhit** [comment]
For the record: this makes playing a synth with these mechanics will make you absolutely awful. It's not extremely obvious when someone is playing with these mechanics (quirks only available with a mediscanner when most doctors just check damage, especially when people are just checking bruises/burns).
The mechanics are pretty painful, as many people have pointed out (and why this PR is there).
So what's going to happen is nobody wants to take the quirk because it's painful. 
Then in 6 months someone will ask "why is this here", and then remove the system in the first place.

This isn't a fix, this doesn't improve anything, it's just a way to push it to the bottom of the barrel and rot.

**Comment by YakumoChen** [comment]
> 
> 
> This is awesome. I understand there was a LOT of salt in the discord about this PR for...some reason but. Ultimately, this helps avoid a LOT of headaches for players and developers alike. Players can pick which form they want and play the game either as a somewhat modified human, or a fullon legacy synthetic with all the experimental features that developers are free to play with and tinker as they see fit and wind up better equipped to receive feedback since players are actually able to make an educated decision of what they're getting into regarding the trait. Also
> 
> For the trait text: Can you add that since this code is highly experimental, features can potentially change quite frequently? Helps give more information to the player

I'm pretty sure it was just you bitching about it endlessly, and not a lot of others.

**Comment by CameronWoof** [comment]
If this PR has committed any crimes at all, it's potentially exacerbating how impossible it is to tell what category (organic/robotic/slime) a given unconscious snowflake in a spacesuit might be.

For ease of gameplay I think that you just need to be able to tell with an examine. Custom options and clothes be damned, just tell me what this fucker IS so I can get him to medbay/robotics/an airlock, respectively without having to make a day of it.

**Comment by Tupinambis** [comment]
This just indicates to me that synths are not fun to play and have inherent design issues, which after reading the documents pertaining to them and talking to people seems to very much be the case. 

Synths should be a proper sidegrade and feel like you are a robot. The issues I have are as follows  (assuming the information I have is current), along with how I'd fix them:

- Synths are hurt by damaging chems but not healing chems. They shouldn't take damage from any internally processed chems period. Stuff like acid would make sense though.
- Difficulty in self healing. As far as I can tell self healing takes longer, and requires tools that are a tad less accessible and take more inventory generally. I think self healing should be equivalent in speed and strength to organic methods, if not a bit stronger to account for the increased scarcity of materials. Of course this is not nearly as big of a deal as...
- Damage thresholds. A huge issue here. You should never need surgery for severe damage unless you're critical. It should be an option and there should always be alternatives. Additionally some of the thresholds and mindamage numbers feel too far on the low side. If a measely laser gun can cause permanant damage that only surgery can fix in just two shots, something is wrong. (and honestly even a couple of stechkin shots still feels excessive.)
- Radiation weakness. This is nonsensical. Robots are not exactly known for going on the frits from radiation. In fact they are more well known for being sent into dangerous areas that humans cannot handle, such as irradiated areas. They should be just immune.
- Why do synths have nutrition, just call it charge full stop and have it so that synths function fully until they run out of charge, at which point they have a short backup battery timespan to either replace their battery or charge it before maybe being super slowed down and no longer able to use their hands? effectively helpless but still able to call for help or find help.
- Corruption: Not sure enough on this, but depends on how it's done.

Of course these fixes don't solve the key issue, which is that these mechanics don't expressly add anything or feel fun. Making the traits not frustrating and more fitting is a good first step, but I feel like more advanced mechanics are needed to really make synths pop as a unique and compelling option.

Modularity is by far the biggest one for me. Let synths bodymod the hell out of themselves in ways that are much easier for a human to do. Want to just pop in an industrial tool arm? Why not. A gun that draws from your power reserves and is also your hand? Fun. Surgery toolset? Why not. These can be better than doing it by hand at the cost of not having free hands, and you can always... switch hands. Of course in a game like ss13 where anything can happen at any time this is balanced by the fact you could be caught with your pants down... and this drains your power. Let them assimilate into mechs and even cyborgs too. Maybe a nerfed version that lacks the ability to interact with electronics remotely, open all doors, or interact with silicon chat. Heck, why not let synths replace their legs with tank treads.

There is so much to play with when it comes to the very concept of a synthetic/robot person, and a few weak tweaks to a race are not nearly enough to realize that. This PR just feels like an admittance of defeat to that fact, and deciding that "Oh well synth mechanics suck probably always will so let's just keep it as an option." maybe that's the case, but it seems lame to me.

**Comment by MrJWhit** [comment]
The idea is that this PR will allow the mechanics to be worked on and adjusted until... it's decided it's the default I suppose. But in the meantime it will be straight virtually untested code where nobody else wants to work on it. See:
image

**Comment by DeltaFire15** [comment]
While this would indeed lead to interesting interactions, I am afraid this would be mostly making them **horrible** for anyone having to deal with them.
Medbay now instead of having the easy 'oh hey it's a synth / IPC species person, fix them the robo way' to 'oh hey maybe this is a synth maybe it isn't, either way there is *no* way to know in advance. Even worse if it's just a nonmed person trying to help who has no access to tools showing quirks.
 The same applies for quite literally everything, example 'oh hey, this is my target - time to poison them - hey why is nothing happening - oh they picked the perk which there is zero way for me to find out in advance.'

Additionally, the current implementation of synthetics is not exactly flawed in a fundamentally unfixable way (or well, if that was the case, making it a quirk wouldn't be a good idea anyways)
The only current problems of significance regarding them are (Least what I've seen & heard from people, can't mindread unfortunately)
* medbay needs a toolbox mapped in
* there's a few bugs, though I'm fixing them whenever I get notified & can track them to synth code
* thresholds are being somewhat problematic, mostly for combat roles - this is currently already being adressed by PRs
* The effect of EMPs on them might be a bit too much, which is being adeessed & monitored.
* **They need people to learn how to deal with Synths**, which I'm working on making not too difficult by having a up-to-date documentation on the wiki (though it may need a pass to make it a bit more easily understandable)
This PR adresses none of these, instead only making it worse by taking away clear identifiability from synthetics (IPC- / Synthlook is pretty obvious), which assists with the learning process.

Additionally, from what I've heard from people I've spoken to, there isn't exactly that many voices to do something like this, no. Maybe like, two or three people I know of.
Most are just simply wishing for an more easy / nonsurgery requiring way to fix thresholds (PR'd) and weaker EMP effects (PR'd, though might need some further nerfs), aswell as some other minor stuff I'd rather not all list.

Closing statement, this takes a highly unique bunch of mechanics from a species group and makes it instead be a snowflakey guess-in-the-dark nightmare.

**Comment by DeltaFire15** [comment]
> The idea is that this PR will allow the mechanics to be worked on and adjusted until... it's decided it's the default I suppose. But in the meantime it will be straight virtually untested code where nobody else wants to work on it. See:
> image

 Considering I am doing quite alot of PRs on this, and you bet I test the stuff, no, they are not at all untested.
They are more in a state requiring them having been played to get a conscise standpoint of their strengths & weaknesses, to give direction for required changes, which I code (I currently have no less then five open PRs that tweak synth stuff)
Doing this would be a step backward, as it'd add alot more of species mechanics & the fact it isn't always on the species to the balancing problem

**Comment by MrJWhit** [comment]
> Considering I am doing quite alot of PRs on this, and you bet I test the stuff, no, they are not at all untested.

I guess "untested" is wrong, "untuned" is a more right word, which, you even said you're working on it. It's not properly balanced. Putting this on a shelf, and making changes for a perceived balance, when nobody's playing it, isn't going to solve it.

**Comment by DeltaFire15** [comment]
Sadly, while I would love to make changes that immediately are perfectly balanced, one has to take into account players. It's one thing to code a thing, and another to have it actually ingame.

@Tupinambis you make alot of good points here.
The threshold part is already being adressed, both by lowering the penalty and adding a nonsurgical way to fix them (PR is up)
Damage healing itself is far faster on synthetics than on organics, which is why the thresholds were put in place initially, especially since welders & wire is incredibly common

The point about chemdamage is very good, I'm going to look at that in the near future and presumably flag toxins in general (minus acid) as nonprocessing in synths, to go with their tox immunity. Other chems I'll have to think about, though they'll likely not full-on lose chemprocessing in the near future.

Rad weakness is something I'd rather keep, especially since there's a upgrade that turns it into a strength, aswell as the fact synths as of now *only* suffer corruption from rads - no mutations, hardstun vomiting, etc etc etc.

Renaming nutrition to charge would probably be neat, though I'd rather keep the effects to be simillar to nutrition - Unless people would like that

Modularity would be something I do have a few ideas for already, but first I'd like finishing work on stomping out general issues before I can work on fun stuff.

**Comment by MrJWhit** [comment]
I'm not saying you're wrong about not having perfect balance.
I'm saying this PR will mean any changes will be forever imbalanced because nobody is going to opt into it outside of one round and then turn it off because medbay will leave them dying outside of medbay because they don't have the proper setup to handle them. Or drop them off to robotics who aren't prepared for people.

**Comment by ghost** [comment]
Doctors hate him. Learn how he ruined all medbay players' lives with one weird trick!

**Comment by CameronWoof** [comment]
image

lmao this post aged like milk in under a minute

**Comment by deathride58** [comment]
Quick Q&A

>Who asked for this?


Due to the harassment that those who've vocally shown support for this PR have received thus far, I will not be revealing the identities of those who've expressed support for this PR's creation. However, they are free to speak up on this PR if they so choose.

>How do you differentiate synths from non-synths?


I was operating under the assumption that medhuds showed the verbose quirk text at all times when examining, but since I don't use medhuds at all, I had no idea that isn't the case at all. That's my bad there, yeah.
I pushed a commit that adds a notice of legacy synth status to both standard med scanners and standard examine text (examine text gets a "hey, this guy's movement looks artificial. he's probably an old robot!" bit, med scanners get a more verbose "hey, your regular tools won't work on this guy, robotics is much better equipped to help him" bit.)

>Just revert it!!!


No.

Reverting it would take far more effort than the option that this PR has taken, as the code for synths touches quite a few significant bits in human health code. Additionally, reverting it wholesale would just be throwing away months of hard work for little gain at all (as a matter of fact, it'd be fairly counterproductive, as toxicity is a pretty huge issue in this community, especially when it comes to development. reverting it wholesale when the mechanics have had as much genuine effort put into them as they've had, would basically just be sending a message to devs saying "we don't want any form of experimentation here". Considering there's quite a few devs that avoid this place purely because of the sheer toxicity of the community, reverting it wholesale would just be further reason for devs to continue straying away from here. The option this PR takes is much better for everyone's collective sanity, as this instead puts the content in a state where it's far less likely for experimental changes to provoke anger from synth players, which in turn grants a *lot* more freedom for development.)

As an aside, I'm personally not in support of the racial traits being a part of standard IPCs/synths as a default thing at all, as the species in question were originally implemented purely for aesthetics alone. It's simply not fair for players at all to implement something that's intended to be purely a cosmetic option, and then later turn around and make that option be a radically different gameplay experience (and I have made repeated promises that any mechanical effects would be implemented as quirks instead of being inherent to the species. The current implementation of synths and IPCs directly violates those promises). Instead, I'd be much more interested in encouraging development to go towards furthering the divide between gameplay for baseline humans, and gameplay for legacy synthetics (i have proposed numerous ideas in the past for individual features that could be implemented to achieve this). By making legacy synthetics be far more interesting to play, this content would be advertised to players not by saying "hey if you wanna be a cute-lookin' robot, you have to be subject to these completely different mechanics", but by rather, by saying "hey, if you feel like the gameplay's pretty stale, here's a little button you can press. it gives you a completely different mechanical ruleset that makes your character play like a walkin' talkin' toaster"

>Why not make plasmamen, podpeople, slimes, etc, into traits as well???


I'm in support of that, yes. But with the way those species are currently handled, turning those things into opt-in quirks would require a massive refactor of the quirk backend; it'd legitimately be easier to rewrite the species system in it's entirety than it'd be to actually turn those species' associated mechanics into opt-in quirks. Ideally, the core of a species (all of the associated mechanical effects, etc), should be completely decoupled from its actual physical appearance, and players should have the freedom to combine wholesale species mechanics where reasonable, so long as the mechanics do not directly conflict with one another (if a player wants to be a mushroom plasman slimeperson plantperson ipc, they should be 100% free to subject themselves to that kind of hell). However, this would simply be impossible without significant heavy lifting.

**Comment by ghost** [comment]
Why can't we just take the obvious route and make synths easier to heal?

**Comment by deathride58** [comment]
> 
> 
> Why can't we just take the obvious route and make synths easier to heal?

That's already a PR (which is compatible with this PR, as an aside). Doesnt fix the core issue at hand, which is that players are pissed about having to be subject to vastly different mechanics in order to play a species that was originally intended to be cosmetic-only.

**Comment by YakumoChen** [comment]
Would be massively against making species traits into quirks. That would utterly ruin the medbay experience for basically everyone. At least the visual appearance of someone gives me an idea of what their needs are, if not their name. Having a slime die from toxin because he took the "lol flesh man" trait would make it abysmally shitty to treat and obstuficates an already under-appreciated job. 

If I can't tell at a glance what kind of treatment a player needs (less of a glance than reading the fine print on a medbay analyzer) when playing doctor it's going to lead to mistakes or just prolong the process, which makes medbay more chaotic and prevents me from giving people timely treatment. Skyrat made it a quirk and made medbay into a game of russian roulette when a corpse got dragged in. Get halfway into prepping the treatment only to realize "oh they're synth, never mind that". and tossing the body out back to the hallway.

Make species traits either all or nothing. Don't make it unfun to play because someone arbitrarily decides they don't want to play with game mechanics in a videogame.

**Comment by timothyteakettle** [comment]
personally like this PR, I wish synths were not as ass in terms of balance, but at the end of the day i'd rather people be able to have fun playing the game, and ultimately this PR will allow synth players who dislike the mechanics to continue having fun

**Comment by R3dtail** [comment]
from an anonymouse synth player:  In my experience playing as a synth, I find many of the aspects quite fascinating and a shake-up from the usual organic playstyle. It makes combat more difficult, sure especially since a single 1x1 emp is enough to nearly end you if you're not quick enough, but at the same time it adds a nice little bit of a dynamic to combat and gives a reason for people to start using lesser-bought abilities/items if they know they're going against a synthetic. Conversely though as it stands, there are a fair few things that need addressing to be playable, such as certain surgeries needing to be added (like embed removal) and addressing organ decay. Additionally, radiation feels like it's extremely detrimental since healing chems cannot be used fix corruption and requires you to hang around in medbay or the roboticist's area until you stop taking corruption since there isn't any other way to purge the stacking corruption you take.

I'm neither for nor against the PR, but I do believe synthetic should stay as a racial, the same way others differ slightly. Either way though, synthetics do have some issues that were raised, like healing chems being nearly non-existant (rad-reduction works for instance) but there are no actual chems that heal a synth. Conversely, effects from toxins like curare (stamina loss and knockout) still affect synthetics which seems quite strange. So if anything, synthetics could greatly benefit from something that lets them get healing even if it's at a reduced rate like tricordrazine. Maybe even an ability that lets them purge corruption themself, but the trade off being they become unconscious for 30 seconds or more while doing so, allowing traitor synths to effectively deal with buildup. In all though, I personally find synths enjoyable to play as they are besides the few gripes listed; of which, many have PRs up for fixing/changing.

**Comment by TripleShades** [comment]
Honestly, I don't even know where to start, except that it just would have been easier to enable antennae on anthromorphs, since that was the only thing missing to make Synthlizards on anthromorph, as I'm pretty sure people who play IPC dont mind that they're playing with robotic mechanics.

Turning an entire species' mechanic into a quirk sets the precedence for making all species unique traits into quirks, which while not a terrible idea on the surface, will later likely introduce upkeep problems, such as having to make new sprites for everything to be compatible. Not to mention that doing this, regardless of the health analyzer changes in this pr, make it an annoyance at best and catastrophic at worst for those who need to perform medical on whoever has the trait, since it clogs up the screen with More Shit that a person needs to scan through before finding out that they can treat them or not (although personally I think this isn't a terrible idea, just having the health scanners put ORGANIC, ROBOTIC, SLIME, etc next to the Base Species: <species> output is largely more readible in hectic situations and applies to more then just synthetic/ipc crew.)

Not only this, but leaving it as an opt-in means that feedback for it will drop to nothing, which is... very poor for the future of maintaining such mechanics, as well as possible balance attempts, since you cant concretely deal with "well this race has x trait which means we can do this and that".

I understand that this wasn't meant to be a flat fix for the current issue of Synthetics/IPCs being hard to deal with medically, but this is honestly not the way to go, and it would be easier and faster to just, as I said before, enable antennae for the regular anthromorph race and fill in your custom species. Having just tested what it looks like, the current health scanner outside of this pr is more than adequate even if you do so, and people have always had to deal with double-checking for species when doing medical even outside of the visual of a character, especially in extreme cases such as husking where it's impossible to identify someone by species without a scan. In addition, this doesn't stop people from roleplaying as more organic-based robotic organisms, as... this is basically what we did anyways before, so if this is really just a cosmetics conflicting with mechanics issue, my proposed solution should be no issue, and allows the continued development of actual robotic crew (synthetics/ipcs) without causing a huge fuss over everything else mentioned above.

I get it, I understand that synthetics/IPCs right now are extremely hard to deal with for both those who play them and those who need to give them medical treatment, but as someone who mains a synthetic, and has always played IPC/synthliz before synthetic anthromorph, the continue development of their mechanics can happen without this extreme pr, and those who dislike them enough to be distraught can still be their favorite species visually without the mechanics. People are learning, willing to learn how to deal with these new developing mechanics. Things can and will become better, it just needs time and... a few small bandaids until it heals into something good, or at least tolerable.

**Comment by deathride58** [comment]
>Would be easier to enable antennae on anthro


Synths have a slimmer head shape, meaning most synth antennae would end up looking misplaced on the standard spaceman head that anthros have. It's only the thicker antennae sprites that'd look right. Only way to fix this would be to allow different species base sprites to be chosen in chargen, which would require some heavy lifting to achieve.

>Med scanner text is cluttered


Yeah, that's admittedly something that's been an issue. Two potential ways I could improve this in the PR would be:

- Make the notice of legacy synthetic status in med scanners have a different font color. This would make it far easier to notice at a glance, as the different font will immediately draw the player's eye to the notice if they're just expecting the standard med scanner output.
- Split the notice up into two sections: "Treatment Complications", and "Recommended Department Involvement". The former would list potential complications in treatment on a single line, such as "Requires plasma chamber for cloning", "Inverted toxin damage, Cryogenic sensitivity", and "Requires industrial tools for repairs". Additionally, the former would only appear if treatment complications are actually present. The latter would state "Medical" by default, but specific cases would modify this, such as plasmamen appending "+ Engineering (For cloning)", and legacy synths replacing the medical portion with "Robotics".

These two things would make the med scanner text be a lot faster to read at a glance (especially if treatment complications are color-coded, though that'd have to be treated with care to make sure it's color-coded in a colorblind-friendly way), on top of making it far easier to fit important information about treatment modifiers into the med scanner text. This won't fix the issue of players just blindly jamming bodies into sleepers, though. That behavior is a problematic habit that simply cannot be resolved with code alone.

One way to somewhat alleviate the issue of doctors just jamming bodies into sleepers blindly, would be to add treatment complication notices to sleeper status text. However, since most players that follow the habit don't even pay attention to the sleeper text at all (literally 90% of the time it happens, they just press every single button blindly and pop the body right back out regardless of the displayed status. There's no thought involved at all), implementing this would probably have no impact whatsoever, all things considered.

EDIT: here's a screenshot of how the above looks ingame
image


>This will ruin the medbay experience!


Dragging someone doesn't stop you from using items. It's more than plausible to slap someone with a med scanner while you're in the middle of dragging someone. And that's a pretty easy habit to form. With the above proposed changes to this PR, this habit would be even more useful to have.

**Comment by TripleShades** [comment]
Hey yeah I actually already tested the antennae enabled on anthro because I was going to make a salt pr that was my suggestions, but couldn't get the scanner to work how I said so w/e.
They look fine. You're being too picky.

I didn't say it would ruin the medical experience, and actually noted that people are actively willing to learn what to do. If anything, we still don't have a solid answer of exactly where they belong to be dragged to right now, since neither medbay nor robotics is outfitted shift-start to deal with them.

I do really enjoy the color-coding you applied to the scanner, but just seeing how long that species name gets is... really gross and easy to just pass your eyes over it without reading. If it were possible to color code each piece?
But wait, noticing your example, then how are slime synthetics going to be treated? Are you trying to do a medical rework too??

It really just sounds like you would rather go ahead despite the overwhelming negative feedback, if I'm honest, and it's only going to make you more upset that work was done for nothing. Sunk cost fallacy and all that.

**Comment by YakumoChen** [comment]
That med scan is already a giant, ugly block of text and that doesn't even include a vitals readout. We need less erroneous text on the readout, not more.

 Look how much information is blasted at you in that healthy specimen.  Now toss him into space next to the active turbine for 3 minutes, after hitting him with 18 laser shots, and show me another readout. I have to look at all of that shit before I treat them, with a clock ticking on a timer for how fucked he'll be by the time his heart is beating again, if I can manage to even get that working, and I could have 3 more bodies in a queue besides.

**Comment by ghost** [comment]
Why is this still open?

**Comment by RogueSteampunker** [comment]
Considering the precedent and the confusion the change would create, I'm going to have to lean towards disagreeing with the PR.

It fails to acknowledge that synths are still new as can be and in need of tweaking, and attempts to sidestep that issue by creating a divide on a radically different species in comparison to humans. Due to this, it's expected to cause medical and those attempting to heal others in emergencies no end of grief at attempting to find the right method to heal synths, in a situation where we already need to put focus on addressing the balance between ease and complexity of the process as is.

Not to mention that, by merging the PR and separating key species attributes into a single quirk, we will be setting the precedent that any other species should be given an opt-in. And between species like mushrooms being fundamentally broken in regards to sprite to have their features removes, to even more situations of confusion and unperceivable factors for crew attempting to combat or heal in regards to those like slimepeople, we'd be leading ourselves to an untenable situations where we'll either be sacrificing the readability of sprites and crewmembers for those involved in the round, or creating a double-standard between synths and every other species.

**Comment by silicons** [comment]
I'll be calling a proper vote on this by the end of this week, with my push being to close this.
This has unironically burned out someone who's been working on synths for a long time and frankly, if people want to be reskinned humans so much, Teak has added a custom species field for just that. There is no need to kill any drive to improve a species because people want to be synths-but-not.

**Comment by silicons** [comment]
see today's discord discussion in coderbus
tl;dr this is being actively worked on let's not toss changes if all else fails we'll see then, plus a few months is enough for us to add the sprites to other species if needed

**Comment by ghost** [comment]
oH MY **S U C C** ING GOD FINALLY
